.github/workflows/gradle.yml

```diff
@@ -44,4 +44,4 @@ jobs:
     #     gradle-version: '8.5'
     #
     # - name: Build with Gradle 8.5
-    #   run: gradle build
+    #   run: gradle build
```

Comment:
I am not able to understand what has changed here...

Reply:
This is for CI. We have not changed anything here.